src/webpack-system-register.js

```diff
@@ -131,7 +131,7 @@ WebpackSystemRegister.prototype.apply = function(compiler) {
 		// Based on https://github.com/webpack/webpack/blob/ded70aef28af38d1deb2ac8ce1d4c7550779963f/lib/WebpackSystemRegister.js
 		compilation.plugin("optimize-chunk-assets", (chunks, callback) => {
 			chunks.forEach(chunk => {
-				if (!chunk.initial) {
+				if (!chunk.isInitial()) {
```

Comment:
Same here about trying to keep it backwards compatible. Maybe something like this: `chunk.isInitial ? !chunk.isInitial() : !chunk.initial`?